### ISSUE TITLE
Magenta Improv RNN problem : INFO:tensorflow:Processed 1 inputs total. Produced 0 outputs.

### ISSUE THREAD

**Issue by K2PF** [tensorflow/magenta#1705]
Hi! When I run “Improv RNN”, I could not get any generated music data after training.
It looks like even a training program has not been starting correctly.
I guess the root cause is my training data set cannot produce any out because My terminal said “INFO:tensorflow:Processed 1 inputs total. Produced 0 outputs.”
I have read all of what my terminal said but I could not understand the root cause and take a countermeasure.
So, it would be very appreciated if someone tell me the root cause of this problem and also countermeasure as well.

Version information is as below.
magenta                  1.3.0                       
tensorflow               1.15.2                      

What my terminal said is as below.

-Insert start-
(magenta) MYNAME-no-MacBook-Pro:~ MYNAME$ improv_rnn_create_dataset \
> --config=attention_improv \
> --input=/Users/MYNAME/magenta2/notesequences.tfrecord \
> --output_dir=/Users/MYNAME/magenta2/improv_rnn/sequence_examples \
> --eval_ratio=0.10
WARNING:tensorflow:From /Users/MYNAME/.pyenv/versions/miniconda3-4.3.30/envs/magenta/lib/python3.7/site-packages/magenta/pipelines/pipeline.py:310: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
W0424 21:55:55.763166 4453928384 deprecation.py:323] From /Users/MYNAME/.pyenv/versions/miniconda3-4.3.30/envs/magenta/lib/python3.7/site-packages/magenta/pipelines/pipeline.py:310: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
INFO:tensorflow:

Completed.

I0424 21:55:55.778475 4453928384 pipeline.py:386] 

Completed.

INFO:tensorflow:Processed 1 inputs total. Produced 0 outputs.
I0424 21:55:55.778677 4453928384 pipeline.py:388] Processed 1 inputs total. Produced 0 outputs.
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_coincident_chords: 0
I0424 21:55:55.778778 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_coincident_chords: 0
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_empty_chord_progressions: 2
I0424 21:55:55.778846 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_empty_chord_progressions: 2
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_melodies_discarded_too_few_pitches: 0
I0424 21:55:55.778917 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_melodies_discarded_too_few_pitches: 0
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_melodies_discarded_too_long: 0
I0424 21:55:55.778976 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_melodies_discarded_too_long: 0
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_melodies_discarded_too_short: 0
I0424 21:55:55.779045 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_melodies_discarded_too_short: 0
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_melodies_truncated: 0
I0424 21:55:55.779103 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_melodies_truncated: 0
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_melody_lengths_in_bars:
  [8,10): 1
  [20,30): 1
I0424 21:55:55.779184 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_melody_lengths_in_bars:
  [8,10): 1
  [20,30): 1
INFO:tensorflow:DAGPipeline_LeadSheetExtractor_training_polyphonic_tracks_discarded: 0
I0424 21:55:55.779243 4453928384 statistics.py:141] DAGPipeline_LeadSheetExtractor_training_polyphonic_tracks_discarded: 0
INFO:tensorflow:DAGPipeline_RandomPartition_training_lead_sheets_count: 1
I0424 21:55:55.779313 4453928384 statistics.py:141] DAGPipeline_RandomPartition_training_lead_sheets_count: 1

-Insert End-

**Comment by envilk** [comment]
Maybe I'm too late but in #1602 appears the same issue, with a different metric ("tracks_discarded_more_than_1_program") and a different model (Polyphony RNN).
